### ISSUE TITLE
Redesign of YDKJS Exercises

### ISSUE THREAD

**Issue by harshPPatel** [austintackaberry/ydkjs-exercises#242]
Hi team,

The project and work is great! That's why I decided to redesign the website. Here is my proposal for a new design for the website.

Figma Link: https://www.figma.com/file/lFqOlIZpOEWVqi8NeOCjPG/YDKJS-Exercises?node-id=0%3A1

I can work on redesign the project once it is approved and everyone is agreed with it. I will be writing most of the css as global CSS because of repetitive code in the project. I will still use some specific component styles in few areas like position, margins etc.

Also we can us the icon as favicon as well (Again If everyone agrees!).

My future plan:
I am also planning to work on Dark Mode as it is trending in the 2020.

There is another idea for this project (I might not work on this but it will be a good update). We should create question for V2 of YDKJS book. Then we can simply have one dropdown where user can choose which book he wants to practice. (Let me know if this is a good idea, if it is then I will create another issue for this)

**Comment by austintackaberry** [comment]
Hey @harshPPatel, these designs look awesome!!!

Feel free to do whatever you want with the css, I have no strong opinions. Perhaps it would be best to create a `redesign` branch and then we can merge it in to master at the end when it's ready?

I don't have a ton of time at the moment to help code this..are you looking for implementation help or do you want to do it by yourself?

**Comment by harshPPatel** [comment]
Hey Austin, thanks for response.

Actually I have already started working on redesign on forked repo. I might change some code as we can animate blocks easily. It will not be any breaking changes though. I will test all the aspects of website before I make a PR with these changes!

If i need any help to understand code, i will drop message in discord.

Happy to be a part of an open source community 😊!

**Comment by austintackaberry** [comment]
Sounds good! Leaving questions/comments/concerns on Discord would be perfect

**Comment by matthewharwood** [comment]
@harshPPatel Could you open up for comments? I'd love to leave some design feedback

**Comment by harshPPatel** [comment]
Hi @matthewharwood , You can drop comments here or inside the Figma file I shared with the issue! All the feedback is appreciated!

**Comment by harshPPatel** [comment]
Hey @matthewharwood , I looked at your comments on Figma. I was thinking that we can work on this together. Any thoughts?

Also as I mentioned before, my discord username is `harsh_patel`, just send me DM if you want to work on this together!

**Comment by matthewharwood** [comment]
@harshPPatel I couldn't find you 
You can join my discord if you wanna talk https://discord.gg/RXJeMS

I had fun and gave a shot at a redesign too: 
https://www.figma.com/file/LA21pC9FqsSabwBC7AAI7S/Design?node-id=1%3A19

**Comment by hoehooiyan** [comment]
Hey @harshPPatel @matthewharwood, are you guys working on the redesign?

**Comment by anthonygedeon** [comment]
Is anybody actively coding the redesign? If not I can take a stab at it.

**Comment by harshPPatel** [comment]
Hey @anthonygedeon , @hoehooiyan . I am not working on this actively. Feel free to work on it as you wish.

**Comment by anthonygedeon** [comment]
does anybody want to pair program for the redesign? I've made minimal progress as of now.

![image](https://user-images.githubusercontent.com/47023801/93400454-71914880-f845-11ea-8eef-c11d7c4efafa.png)

**Comment by matthewharwood** [comment]
@anthonygedeon are you using my design? https://www.figma.com/file/LA21pC9FqsSabwBC7AAI7S/Design?node-id=14%3A2050 I think @harshPPatel and I agree'd it was a better starting place.

Unfortunately I don't have the bandwidth to help program right now :(

I fyou have any design questions go for it tho.

**Comment by anthonygedeon** [comment]
Thanks! I'll make sure to ask questions.

**Comment by hoehooiyan** [comment]
> does anybody want to pair program for the redesign? I've made minimal progress as of now.
> 
> ![image](https://user-images.githubusercontent.com/47023801/93400454-71914880-f845-11ea-8eef-c11d7c4efafa.png)

Hey Anthony, maybe we can do it together, how do we connect with each other for the discussion?

**Comment by austintackaberry** [comment]
You both can use the discord channel if you want!
